spec/factories/broadcasts.rb

```diff
@@ -22,13 +22,11 @@
       end
     end
 
-    factory :twitter_connect_broadcast do
-      title          { "Welcome Notification: twitter_connect" }
+    factory :apple_connect_broadcast do
+      title          { "Welcome Notification: apple_connect" }
       type_of        { "Welcome" }
       processed_html do
-        "You're on a roll! 🎉 Do you have a Twitter account? Consider " \
-        "<a href='/settings'>connecting it</a> so we can @mention you if we share " \
-        "your post via our Twitter account <a href='https://twitter.com/thePracticalDev'>@thePracticalDev</a>."
+        "You're on a roll! 🎉  Do you have an Apple account? Consider <a href='/settings'>connecting it</a>."
```

Comment:
I think this broadcast may need to be added to a) seeds.rb, and b) a data migration to add it to the database!

Reply:
Here's what I came up with for the Facebook auth PR: https://github.com/forem/forem/commit/65de0e7b408f55726da0dc271c80a686b628d342